apps/docs/pages/docs/api-reference/components/puck.mdx

```diff
@@ -349,3 +350,22 @@ By default, Puck exposes small, medium and large viewports based on common viewp
   }
 ]
 ```
+
+### `initialHistories`
```

Comment:
Same goes for this - the content is alphabetical, so this should move underneath the `iframe` section.

Reply:
You moved this under onChange instead of iframe by mistake - mind bumping it up one more? Thanks!

Reply:
Woops. Fixed.